tests/utils/test_utils.py

```diff
@@ -33,3 +33,21 @@ def test_get_change_url_no_git_release(mock_repo):
 def test_get_change_url_with_git_release(mock_repo, mock_git_release):
     url = get_change_url(tag_name="v2.0.0", repository=mock_repo, git_release=mock_git_release)
     assert url == "https://github.com/org/repo/compare/v1.0.0...v2.0.0"
+
+
+# detect_row_format_invalid_keywords
+
+
+def test_valid_row_format():
+    """Test when all keywords in the row format are valid."""
```

Comment:
You dont use method docstring anywhere else in the test file. Please choose and keep the same pattern.

Reply:
Removed in commit aee3c5f4428a5d0b27b8f44dad6241a36ea8f7d6.